### ISSUE TITLE
Project Feedback!

### ISSUE THREAD

**Issue by codepathreview** [tylerw909/Finstagram#2]
It looks like the following features are not reflected on your GIF walkthrough: 

- User can post the photo with caption to the server.

In order for us to count these towards your submission, please record another GIF that captures these features. Once you do, please push your updates and **submit your assignment again through the course portal within 48 hours from the posted deadline so that we can regrade it**.

**Comment by tylerw909** [comment]
Yes, I did. The most recent post in the video is the one I just added. Username: "test" Caption: "new year new post". It is with the waterfall image.
